Sources/UINavigationController.swift

```diff
@@ -62,6 +65,11 @@ open class UINavigationController: UIViewController {
     }
 
     open override func dismiss(animated: Bool, completion: (() -> Void)? = nil) {
+        if self === UIApplication.shared.keyWindow?.rootViewController {
+            completion?()
+            return
+        }
```

Comment:
@rikner this should fix the issue you were seeing

Reply:
I tested this in the Mac App (Esc has the same effect as the physical back button on Android) and it works as expected (as in, it doesn't do anything) 👍🏼

Reply:
yes, can confirm 👍